astropy/coordinates/builtin_frames/galactocentric.py

```diff
@@ -168,14 +223,40 @@ def validate(cls, value):
             attrs = dict()
             for k in value.frame_attributes:
                 attrs[k] = getattr(value, k)
-            cls._references = value.frame_attribute_references()
+            cls._references = value.frame_attribute_references.copy()
+            cls._state = dict()
             return attrs
 
         else:
             raise ValueError("Invalid input to retrieve solar parameters for "
                              "Galactocentric frame: input must be a string, "
                              "dict, or Galactocentric instance")
 
+    @classmethod
+    def register(
+        cls,
+        name: str,
+        value: dict,
+        references=None,
+        **state: dict,
```

Comment:
Since this is not what we actually return as `state`, would it make sense to call this `meta`? And then, of course, below
```
state = dict(value=value, references=references)
state.update(meta)
```

Reply:
- [ ] Implemented